code/modules/projectiles/projectile/beams.dm

```diff
@@ -88,6 +140,22 @@
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/green_laser
 	light_color = LIGHT_COLOR_GREEN
 
+/obj/item/projectile/beam/emitter/hitscan
```

Comment:
This appears to be a duplicate of whats in special_projectiles.dm at line 286

Reply:
duplicate removed